doc/neptune-metadata.rst

```diff
@@ -84,7 +84,9 @@ Type
 Meaning
   Defines which types of auto-pull-requests the repository desires.
   Values can be specified for ``default`` and ``override``.
-  Possible values are ``none``, ``patch``, ``minor``, ``major``
+  Possible values are ``none``, ``patch``, ``minor``, ``major``.
+  The default value depends. If it is a library repository (``library: true``)
+  the default is ``patch``. Otherwise it will be ``minor``.
```

Comment:
Can I ask the reason for this discrepancy?  Is the assumption that non-library submodules will be things like makd or beaver (i.e. build/CI tools) ... ?

Reply:
Generally, libraries should have as old dependencies as possible because if a library has a newer minor branch as dependency that it doesn't actually need, it means programs that use it also must use that newer library.

Reply:
Oh, sorry!  I mixed up the meaning here.  So if the _receiving_ project is a library, its automatic PRs will only bump patch versions.

I misinterpreted this as meaning that if the submodule was a library, the receiving project would only get patch updates.